src/Simulation.cpp

```diff
@@ -945,7 +945,8 @@ void Simulation::simulate() {
 	Timer perStepTimer;
 	perStepTimer.reset();
 
-	_simstep = _initSimulation + 1;
+	// keppRunning increments the simstep counter before the first iteration
```

Comment:
change "keppRunning" -> "keepRunning()" - typo and add parentheses

Reply:
Changed 43f9e95